routemaster/context.py

```diff
@@ -19,6 +19,7 @@ def __init__(
         now: datetime.datetime = None,
         feeds: Dict[str, Feed] = {},
         accessed_variables: Iterable[str] = [],
+        current_history_entry: Optional[Any] = None,
```

Comment:
Do we really not have a type for the entry? Or is it that SqlAlchemy doesn't provide a useful type?

Reply:
The latter, because we aren't using the ORM. This will change eventually I hope.